packages/react-mutation-mapper/src/component/revue/Revue.tsx

```diff
@@ -34,15 +34,18 @@ export const RevueTooltipContent: React.FunctionComponent<{
                     </a>
                 }
                 {` (`}
-                {
-                    <a
-                        href={`https://pubmed.ncbi.nlm.nih.gov/${props.vue.pubmedId}/`}
-                        rel="noopener noreferrer"
-                        target="_blank"
-                    >
-                        {props.vue.referenceText}
-                    </a>
-                }
+                {props.vue.references.map((reference, index) => (
+                    <span key={index}>
+                        <a
+                            href={`https://pubmed.ncbi.nlm.nih.gov/${reference.pubmedId}/`}
+                            rel="noopener noreferrer"
+                            target="_blank"
+                        >
+                            {reference.referenceText}
+                        </a>
+                        {index !== props.vue.references.length - 1 ? ';' : ''}
```

Comment:
This would be slightly more readable. Not a big deal though.

```suggestion
                        {index < props.vue.references.length && ';'}
```

Reply:
I change it to `{index < props.vue.references.length - 1 && ';'}`, no ';' adding when it has only one reference